### PR TITLE
`setup-kind-cluster`

### PR DESCRIPTION
**Pull request by mathieu-benoit**
- `setup-kind-cluster`
- `alpine:3.20.0`
- Apache License
- Simplify docs with `make compose-test` and `make k8s-test`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM alpine:3.19.1 as builder
+FROM alpine:3.20.0 as builder
```
Comment (astromechza) [inline Dockerfile]: To reduce maintenance burden, you should be able to just go with 3.20 or even alpine:3

File: LICENSE [hunk 2]
```diff
@@ -1,19 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (astromechza) [inline LICENSE]: may need to put 2024 Platco/Humanitec here?

**Review by astromechza — APPROVED** [review]
LGTM - some nits/comments

File: LICENSE (hunk 2 above)
Comment (mathieu-benoit) [reply LICENSE]: Good catch, fixed. Took this from there https://github.com/score-spec/sample-app-gif/pull/1, fixing it there too, JFYI. I took what we have for now in `score-k8s` as an example.

File: Dockerfile (hunk 1 above)
Comment (mathieu-benoit) [reply Dockerfile]: Sure, not a good practice to show, but for this purpose, agree with you, let's do this. Fixed.

**Pull request merged**